src/test/java/de/retest/suite/flow/CreateChangesetForAllDiffrencesFlowTest.java

```diff
@@ -98,11 +98,11 @@ public void apply_should_throw_exception_whitout_the_stateFilePath() throws Exce
 
 		assertThatThrownBy( () -> ApplyChangesToStatesFlow.apply( persistence, changesWhitoutStateFilePath ) ) //
 				.isInstanceOf( NoGoldenMasterFoundException.class ) //
-				.hasMessage( "No Golden Master file(s) with the following name(s) found:\n\tnull" );
+				.hasMessage( "The following Golden Master(s) cannot be found:\n\t- null" );
```

Comment:
Shouldn't `ArrayUtils` prevent this from being null?

Reply:
There is a difference when passing nothing (i.e. `null`) or the Array `[ null ]`